### PR TITLE
Fix grammar in implementation docs

### PR DESCRIPTION
**Pull request by jwakely**

### CODE REVIEW

**Comment by trapas** [comment]
Can one of the admins verify this patch?

**Comment by mkutlak** [comment]
add to whitelist

File: doc/implementation [hunk 1]
```diff
@@ -55,35 +55,35 @@ daemon process the written info and saves some aditional information:
...
+* UUID is a unique id of the crash in the database, every user is allowed to see only
+  their own crashes or kerneloops crashes.
+- See http://git.fedorahosted.org/git/abrt.git?p=abrt.git;a=blob;f=lib/Plugins/SQLite3.cpp line 394
+  for more details.
 
 1. processing coredump:
     a) determine if the crashed binary belongs to some installed package
     if it does then:
-    b) daemon tries to install the debuginfo issuing this command:
+    b) daemon tries to install the debuginfo issuing by this command:
```
Comment (ernestask) [inline doc/implementation]: “by issuing”

Comment (jwakely) [reply doc/implementation]: done

**Comment by jwakely** [comment]
@michalfabik done

**Pull request merged**